### DISCUSSION TITLE
Disable/Sticky HUD

### DISCUSSION THREAD

**Discussion by MeusRex** [MeusRex/pf2e-chex#20] [Ideas]
When drawing the HUD can be supremely annoying. It should either be hideable, or there should be an option to make it sticky, so that it can be placed at a corner of the screen.

Also, the HUD should scale to the zoom factor. Currently, when zoomed in, it fills the entire screen.

**Comment by federicoacker** [comment]
is this about the window that opens when hovering over a tile?

**Reply by MeusRex** [reply]
Yes, while preparing for one of my campaignes last sunday I found that the hex information "popup" can get in the way. Especially when painting from left to right, or when looking for a hex with a specific feature. So, while painting this popup should probably not show anymore.

**Reply by federicoacker** [reply]
mhhhhh yeah I've had a similar experience, I think maybe adding a toggle button to disable it showing and adding a scale setting in the module settings would make it better, otherwise yeah just making it auto-scale with zoom and having a toggle button is more than enough
